keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1562,8 +1937,17 @@ details). For now, we leave it here.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+No change from existing behavior of the Job controller.
+
 ###### What are other known failure modes?
 
+  - When `PodDIsruptionConditions` enabled pods are not terminated based on the NoExecute taint
+    - Versions affected: 1.25.0 - 1.25.2
```

Comment:
Given the discussion in the cherry-pick, I'd just call out that it's a known alpha bug in 1.25.x

Reply:
Updated